include/oneapi/dpl/pstl/iterator_defs.h

```diff
@@ -67,8 +67,8 @@ struct __is_random_access_iterator_impl<_IteratorType,
 /* iterator */
 template <typename _IteratorType, typename... _OtherIteratorTypes>
 struct __is_random_access_iterator
```

Comment:
to be honest, I would reimplement this trait keeping in mind that we have C++17. ``std::conjunction`` fits better. Furthermore, with std::conjunction it should not be a `struct` anymore. template alias is sufficient.

Also the specialization for one type goes away. If you consider this change risky for the current PR, I am fine with that.

Reply:
Not in this PR.